### ISSUE TITLE
[iOS] Update iOS headers breaks on RN 0.40.0

### ISSUE THREAD

**Issue by HofmannZ** [wkh237/react-native-fetch-blob#244]
Hey could you update the ios headers like [this repo](https://github.com/react-native-community/react-native-blur/commit/19295fcd1d70536d696351f5d5c41782196433a3) did.

They changed it in RN 0.40.0 and brakes all packages with iOS headers 😕.
I suggest you branch of since they did not make it backward compatible so you can still keep support for RN < 0.40.0.

For more info check this: [https://github.com/facebook/react-native/releases/tag/v0.40.0](https://github.com/facebook/react-native/releases/tag/v0.40.0).

**Comment by BretJohnson** [comment]
To fix this and maintain backwards compatibility with RN <= 0.39 without having to branch you can also change your includes like below, using `#if __has_include`. That's what we did for our React Native libraries and it worked well.
```
// Support React Native headers both in the React namespace, where they are in RN version 0.40+,
// and no namespace, for older versions of React Native
#if __has_include(<React/RCTBridge.h>)
#import <React/RCTBridge.h>
#else
#import "RCTBridge.h"
#endif
```

**Comment by fmmajd** [comment]
@BretJohnson where should we add this?

**Comment by BretJohnson** [comment]
See more here: https://github.com/facebook/react-native/releases/tag/v0.40.0.
Basically every react header now has moved into the React namespace.  So every ``#import "RCT..."` you have in your .h and .m files should now change to `#import <React/RCT...>` for RN version 0.40 and above. For RN versions less than 0.40 it should continue to `#import "RCT..."` the old way. And that `#if __has_include(...)` check above can be used to support both versions of RN without branching your code. Here's an example change I made earlier to our stuff for this: https://github.com/Microsoft/mobile-center-sdk-react-native/pull/16/commits/216403641919f8314ad5135fd79c5185626cc928.

**Comment by egunsoma** [comment]
This works for me on local machine, please verify this.

**Comment by wkh237** [comment]
@BretJohnson  and @egunsoma , thanks for the information and PR, looks like this is a great way to solve #223 👍  The PR already merged will publish a new release today 😄

**Comment by wkh237** [comment]
Already published to npm, please upgrade the packge to `0.10.2-beta.7` and verify if it works, thanks !
